modules/ossm-adding-project-using-smm-resource-cli.adoc

```diff
@@ -4,15 +4,15 @@
 
 :_content-type: PROCEDURE
 [id="ossm-adding-project-using-smm-resource-cli_{context}"]
-= Adding a project to the service mesh using the ServiceMeshMember resource with the CLI
+= Adding a project to the mesh using the ServiceMeshMember resource with the CLI
 
-You can add one or more projects to the {SMProductShortName} from the CLI.
+You can add one or more projects to the mesh using the `ServiceMeshMember` resource with the CLI.
 
 .Prerequisites
-* An installed, verified {SMProductName} Operator.
-* Name of the `ServiceMeshControlPlane` resource and the name of the project it belongs to.
-* Name of the project you want to add to the mesh.
-* A service mesh administrator must explicitly grant access to the service mesh. Administrators can grant users permissions to access the mesh by assigning them the `mesh-user` `Role` using a `RoleBinding` or `ClusterRoleBinding`. For more information, see xref:../../service_mesh/v2x/ossm-profiles-users.html#ossm-members_ossm-profiles-users[Creating the {SMProductName} members].
+* You have installed the {SMProductName} Operator.
+* You know the name of the `ServiceMeshControlPlane` resource and the name of the project it belongs to.
+* You know the name of the project you want to add to the mesh.
+* A service mesh administrator must explicitly grant access to the service mesh. Administrators can grant users permissions to access the mesh by assigning them the `mesh-user` `Role` using a `RoleBinding` or `ClusterRoleBinding`. For more information, see Creating the {SMProductName} members.
```

Comment:
```suggestion
* A service mesh administrator must explicitly grant access to the service mesh. Administrators can grant users permissions to access the mesh by assigning them the `mesh-user` `Role` using a `RoleBinding` or `ClusterRoleBinding`. For more information, see _Creating the {SMProductName} members_.
```
I think the title of the cross-referenced section needs to be in italics or quotation marks to set it off from other text. I'm not sure which, but here is the ISG guidance on references: https://www.ibm.com/docs/en/ibm-style?topic=references-citations. I see in italics in some places in d.o.c.

Reply:
done